File: bialydunajec-academic-ministry/application/src/main/kotlin/org/bialydunajec/academicministry/application/dto/PersonalTitleDto.kt
package org.bialydunajec.academicministry.application.dto

import org.bialydunajec.ddd.domain.base.validation.constraints.NullOrNotBlank

data class PersonalTitleDto(
        @field:NullOrNotBlank
        val name: String?,

        @field:NullOrNotBlank
        val prefix: String?,

        @field:NullOrNotBlank
        val postfix: String?
)

File: frontend-adminpanel/projects/bialydunajec-admin/src/app/academic-ministry/service/rest/dto/personal-title.dto.ts
export class PersonalTitleDto {
  name?: string;
  prefix?: string;
  postfix?: string;


  constructor(prefix: string, postfix: string, name: string = null) {
    this.name = name;
    this.prefix = prefix;
    this.postfix = postfix;
  }
}
